app/classifier/world-wide-telescope.jsx

```diff
@@ -480,15 +480,18 @@ export default class WorldWideTelescope extends React.Component {
     }
 
     return (
-      <div>
+      <div className="worldwide-telescope">
         {!!plates.length && (<p>View Your Classification!</p>)}
         {plates.map((plate, idx) => {
           return (
-            <div className="worldwide-telescope" key={idx}>
+            <div className="worldwide-telescope__container" key={idx}>
               <div>
                 <img role="presentation" className="worldwide-telescope__chart-image" src={`${plate.getCropUrl()}`} />
               </div>
-              <a target="_blank" rel="noopener noreferrer" href={plate.getWwtUrl()} className="standard-button">World Wide Telescope</a>
+              <div className="worldwide-telescope__content">
+                <a target="_blank" rel="noopener noreferrer" href={plate.getWwtUrl()} className="standard-button">World Wide Telescope</a>
+                <p>If the image appears misaligned within World Wide Telescope, please click on Talk and label the subject with <a target="_blank" rel="noopener noreferrer" href="https://zooniverse.org/talk/search/?query=misaligned">#misaligned</a>.</p>
```

Comment:
Should this link to the hashtag on the project Talk, instead of top-level Zoo Talk?

Reply:
I can't find the project that's using the WWT component, but links to project hashtags look like this: https://www.zooniverse.org/projects/stephenresearch/beluga-bits/talk/tags/selfie

Reply:
Yes, this could be. I'm initially a bit hesitant to tie a component like this so tightly to a single project, but the use case is very specific to this project so I can make the change.

Reply:
Maybe have this component take a project slug as a prop?

Reply:
Ahh, of course. That makes total sense. I'll push up a commit shortly.